config/initializers/address_lookup.rb

```diff
@@ -4,7 +4,7 @@
   config.address_facade_url     = "/address-service/v1/addresses/postcode"
 
   # 5 was causing many timeouts in development
-  config.timeout_in_seconds     = 10
-  config.address_facade_client_id =  Rails.application.secrets.address_facade_client_id
-  config.address_facade_key       =  Rails.application.secrets.address_facade_key
+  config.timeout_in_seconds       = Rails.application.secrets.address_facade_timeout
+  config.address_facade_client_id = Rails.application.secrets.address_facade_client_id
+  config.address_facade_key       = Rails.application.secrets.address_facade_key
```

Comment:
yeah I agree, it's slightly odd pattern to me, IMHO not sure what secrets is giving us here, why we don't just use the ENV directly in the initializer, given that all the secrets does is assign ENV['ADDRESS_FACADE_TIMEOUT']